### PR TITLE
Update morango and stop locking sync when db backend is postgres

### PR DESCRIPTION
**Pull request by bjester**
## Summary

- Updates morango to `v0.6.14` https://github.com/learningequality/morango/releases/tag/v0.6.14
- Fixes https://github.com/learningequality/kolibri/issues/9524
- Added makefile recipes to facilitate easier testing with postgres

## References

https://github.com/learningequality/kolibri/issues/9524

## Reviewer guidance

- Initiate a sync with the client instance using a postgres backend
- Run sync integration tests

----

## Testing checklist

- [X] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [X] Critical and brittle code paths are covered by unit tests

## PR process

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [X] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by bjester** [comment]
I'm not sure what's run with the linting check

**Comment by rtibbles** [comment]
It's the `git://` protocol issue again - we can go ahead and merge without it.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: '3.4'
+
+services:
+  postgres:
+    image: postgres:12
```
Comment (jredrejo) [inline docker-compose.yml:5]: Taking into account that GCP uses v13 as default, and thinking in the future, wouldn it be best to use postgres:13?

**Review by jredrejo — APPROVED** [review]
I left a small nitpick comment. Also I see this is targeted for v0.14.x. I guess this is going to be ported to v0.15.x and future releases too, isn't it?

File: docker-compose.yml (hunk 1 above)
Comment (rtibbles) [reply docker-compose.yml:5]: Our CI tests currently use 12, so I think good for it to be consistent.

Comment (bjester) [reply docker-compose.yml:5]: Our github actions (in newer release branches) use version 12, so this is consistent with that. Hopefully this shouldn't make much of a difference

**Pull request merged**